templates/go/scaffolding/README.md

```diff
@@ -0,0 +1,10 @@
+# Go Scaffolding
+
+Organized into directories conforming to the naming convention
+[signature]-[invocation] where signature is the method signature group expected
+(static, instanced etc) and the invocation is the hint of what kind of requests
+the function expects.
```

Comment:
This seems like it might be unique to each runtime. For example, with Node.js I don't think separating the instanced path from the static path is necessary. I don't think this is necessarily a problem. Just something to note.